Corgibytes.Freshli.Cli/Functionality/Git/IGitManager.cs

```diff
@@ -0,0 +1,7 @@
+namespace Corgibytes.Freshli.Cli.Functionality.Git;
+
+public interface IGitManager
+{
+    public GitCommitIdentifier ParseCommitSha(string commitSha);
```

Comment:
I think we should stick to calling it either "commit id" or "commit sha". I know they are the same thing but having a different name might confuse someone who doesn't know it's the same.

Reply:
Good point. I'll clean that up.

Reply:
This was taken care of in 12bca43 and 031ee33.